### ISSUE TITLE
Typescript error when using require('emoji-regex')

### ISSUE THREAD

**Issue by rabarrota** [mathiasbynens/emoji-regex#102]
I'm getting the following Intellisense error when using `require('emoji-regex')`

```
This expression is not callable.
  Type 'typeof import("emoji-regex")' has no call signatures.ts(2349)
```

Some context for how I'm using this:
```
const emojiRegex = require('emoji-regex');

...

const sanitizedText = text.replace(emojiRegex(), '');

```

This package works fine this way when actually being used. It just always shows this Typescript error.

<img width="528" alt="Screen Shot 2022-10-07 at 11 15 46" src="https://user-images.githubusercontent.com/104509210/194588615-a6ee2953-dff3-479c-a927-5e6b4d8f04e7.png">

**Comment by chloe-glowy** [comment]
Interesting, I can't repro this, it works fine for me:

<img width="232" alt="image" src="https://user-images.githubusercontent.com/119768192/205469929-31036231-dd8e-4e0e-9eab-9b8189a823f8.png">

Do you think it's maybe because you're using `require()` instead of `import`?

Maybe because the [type declaration module](https://github.com/mathiasbynens/emoji-regex/blob/main/index.d.ts) uses `declare module { }` it only works if you're using module-style imports? 

Also what version of the package are you running?

**Comment by rabarrota** [comment]
Yup, I'm assuming that's the problem. I'm using cjs so it doesn't look like the type declaration supports it.

I'm using `10.2.1`

**Comment by vdh** [comment]
The `index.d.ts` should be updated to this instead:
```diff
diff --git a/index.d.ts b/index.d.ts
index 6f57555090390e09a3b3a2e39bab037c995b47b9..beb73be29022bf20734057e38148282207f5fa2b 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -1,3 +1,4 @@
 declare module 'emoji-regex' {
-  export default function emojiRegex(): RegExp;
+  function emojiRegex(): RegExp;
+  export = emojiRegex;
 }
```
